runtime/core/grpc/wenet.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option java_package = "ex.grpc";
+option objc_class_prefix = "wenet";
+
+package wenet;
+
+service ASR{
+  rpc Recognize(stream Request) returns (stream Response) {}
+}
+
+message Request{
+  int32 nbest = 1;
+  bool continuous_decoding = 2;
+  bytes audio_data = 3;
+}
+
+message Response{
+  string response_json = 1;
```

Comment:
if we use grpc, we should specify all the fields directly here instead of wrap it in the json string, right?

Reply:
We used a more complex proto, which gives explicit definition to all fields now. However, it may cause some inconvenience to folks who are new to protobuf.